monai/transforms/compose.py

```diff
@@ -14,6 +14,7 @@
 
 from __future__ import annotations
 
+import random
```

Comment:
The `SomeOf` class should inherit from `Randomizable` and use our existing methods for doing randomisation, otherwise we're introducing another way of doing random state in addition that found in Numpy and Pytorch. This would make ensuring determinism much harder as well.

Reply:
Thanks. Switched to using self.R instead of a call to the random module.  The `Compose` class already inherits `Randomizable`, and `SomeOf` inherits `Compose`